appveyor.yml

```diff
@@ -8,28 +8,50 @@ environment:
 
   matrix:
 
-    - TOOLCHAIN: "ninja-vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\foo
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-
-    - TOOLCHAIN: "nmake-vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\foo
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-
-    - TOOLCHAIN: "vs-15-2017-win64-cxx17"
-      PROJECT_DIR: examples\foo
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
-
-    - TOOLCHAIN: "vs-14-2015-sdk-8-1"
-      PROJECT_DIR: examples\foo
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-
-    - TOOLCHAIN: "mingw-cxx17"
-      PROJECT_DIR: examples\foo
-      APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
-
-    - TOOLCHAIN: "msys-cxx17"
-      PROJECT_DIR: examples\foo
+    # FIXME: ninja fails to compile Boost
+    # https://ci.appveyor.com/project/kgeorgiev93/hunter/builds/26619052
+    #- TOOLCHAIN: "ninja-vs-15-2017-win64-cxx17"
+      #PROJECT_DIR: examples\ros_comm
+      #APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      #HUNTER_BINARY_DIR: C:\__BIN
+
+    # FIXME: nmake fails to compile Boost
+    # https://ci.appveyor.com/project/kgeorgiev93/hunter/builds/26619052
+    #- TOOLCHAIN: "nmake-vs-15-2017-win64-cxx17"
+      #PROJECT_DIR: examples\ros_comm
+      #APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      #HUNTER_BINARY_DIR: C:\__BIN
+
+    # FIXME: vs-15 fails to compile rospack
+    # https://ci.appveyor.com/project/kgeorgiev93/hunter/builds/26619918
+    #- TOOLCHAIN: "vs-15-2017-win64-cxx17"
+      #PROJECT_DIR: examples\ros_comm
+      #APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+      #HUNTER_BINARY_DIR: C:\__BIN
+
+    # FIXME: vs-14 fails to compile rospack
+    # https://ci.appveyor.com/project/kgeorgiev93/hunter/builds/26619918
+    #- TOOLCHAIN: "vs-14-2015-sdk-8-1"
+      #PROJECT_DIR: examples\ros_comm
+      #APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      #HUNTER_BINARY_DIR: C:\__BIN
+
+    # FIXME: mingw fails to import catkin
+    # https://ci.appveyor.com/project/kgeorgiev93/hunter/builds/26619918
+    #- TOOLCHAIN: "mingw-cxx17"
+      #PROJECT_DIR: examples\ros_comm
+      #APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      #HUNTER_BINARY_DIR: C:\__BIN
+
+    # FIXME: msys fails to compile Boost
+    # https://ci.appveyor.com/project/kgeorgiev93/hunter/builds/26619052
+    #- TOOLCHAIN: "msys-cxx17"
+      #PROJECT_DIR: examples\ros_comm
+      #APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+      #HUNTER_BINARY_DIR: C:\__BIN
+
+    - TOOLCHAIN: "dummy"
```

Comment:
We are not testing anything, both `appveyor.yml` and `.travis.yml` only contains `dummy` toolchain.

Reply:
Can the job timeout be increased? Because the gcc build takes too long, otherwise it will work.

Reply:
> Can the job timeout be increased? Because the gcc build takes too long, otherwise it will work.

I mean the job maximum log length.

Reply:
> I mean the job maximum log length

Try to add `VERBOSE=0`:
- https://docs.hunter.sh/en/latest/creating-new/create/cmake.html#fix-travis-log-too-long-error

Reply:
Of course.. Updated and rebased.